### PR TITLE
docs: create HTML report filtered by `label:chromium-bidi-2023`

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

File: .github/workflows/wpt.yml [hunk 1]
```diff
@@ -57,7 +57,9 @@ jobs:
       - name: Prepare Pages
         run: |
           mkdir -p out/site
+          cp out/wptreport.json out/site/wptreport.json
```
Comment (mathiasbynens) [inline .github/workflows/wpt.yml:60]: Not strictly required, but this lets us more easily test the HTML formatter locally against the latest results (without having to rerun the tests or having to look for the latest artifact download link).

**Comment by mathiasbynens** [comment]
image

File: tools/wpt-report-builder/filter-2023.mjs [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  '/webdriver/tests/bidi/session/status/status.py',
+  '/webdriver/tests/bidi/session/subscribe/contexts.py',
+  '/webdriver/tests/bidi/session/subscribe/events.py',
+  '/webdriver/tests/bidi/session/subscribe/invalid.py',
+  '/webdriver/tests/bidi/session/unsubscribe/contexts.py',
+  '/webdriver/tests/bidi/session/unsubscribe/events.py',
+  '/webdriver/tests/bidi/session/unsubscribe/invalid.py',
+]);
+
+export const apply2013Filter = (reportData) => {
```
Comment (Lightning00Blade) [inline tools/wpt-report-builder/filter-2023.mjs:200]:
```suggestion
export const apply2023Filter = (reportData) => {
```

Comment (mathiasbynens) [reply tools/wpt-report-builder/filter-2023.mjs:200]: Good eye!

Comment (mathiasbynens) [reply tools/wpt-report-builder/filter-2023.mjs]: Changed it elsewhere too.

File: .github/workflows/wpt.yml (hunk 1 above)
Comment (OrKoN) [reply .github/workflows/wpt.yml:60]: "1 conversation must be resolved before merging."

**Pull request merged**